src/components/account-card/__tests__/__snapshots__/account-card.test.tsx.snap

```diff
@@ -6,7 +6,7 @@ exports[`account card component renders correctly 1`] = `
   <View style={{...}}>
     <View style={{...}}>
       <withTheme(TextComponent) format={{...}}>
-        12332
+        0.000
```

Comment:
since the test didn't change, and the balance value is mocked to `12332`, this doesn't seem to be fine...